L/libseat/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libseat"
+version = v"0.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://git.sr.ht/~kennylevinsen/seatd/archive/$version.tar.gz", "274b56324fc81ca6002bc1cdd387668dee34a6e1063e5f3896805c3770948988"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/seatd-*/
+atomic_patch ../patches/fix_crosscompile.patch
+mkdir build
+cd build
+apk add meson scdoc
+
+# pkgconfig returns the wrong path here. It's an ugly hack, but it works
+mkdir -p /workspace/destdir/usr/bin/
+ln -s $(which scdoc) /workspace/destdir/usr/bin/
+
+meson -D werror=false ../ --cross-file="${MESON_TARGET_TOOLCHAIN}"
```

Comment:
```suggestion
PKG_CONFIG_SYSROOT_DIR="" meson -D werror=false ../ --cross-file="${MESON_TARGET_TOOLCHAIN}"
```

Reply:
Does that mean the symlink hack can be removed?

Reply:
Yes